lib/rspec/core/example_group.rb

```diff
@@ -26,7 +26,7 @@ module Core
     # your examples defined in {MemoizedHelpers} and {Pending}.
     class ExampleGroup
       extend Hooks
-
+      extend FailFastHandler
```

Comment:
Actually, seeing `failures_number_met?(reporter.failed_examples.count)` below makes me think that these method defs should perhaps move to the reporter: then it can just be `reporter.fail_fast_limit_met?`.  After all, the reporter holds the state of `reporter.failed_examples.count` already and it has a reference to the configuration instance.  We try to limit how much we access our global state (e.g. `RSpec.configuration` and `RSpec.world`) so it's nice to use instances that objects have already been passed such as reporter already having a config instance.